### PR TITLE
feat(demo): adds dev subcommand for jq selectors to generate and test selectors on a subject context JSON or JWT

### PR DESCRIPTION
**Pull request by jakedoublev**
Related to: https://github.com/opentdf/platform/issues/477

### CODE REVIEW

File: cmd/dev.go [hunk 1]
```diff
@@ -13,6 +13,13 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// devCmd is the command for playground-style development
+var devCmd = &cobra.Command{
+	Use:   man.Docs.GetDoc("dev").Use,
+	Short: man.Docs.GetDoc("dev").Short,
+	Long:  man.Docs.GetDoc("dev").Long,
+}
+
```
Comment (jrschumacher) [inline cmd/dev.go:22]: You can use the GetCommand here so you don't have to map everything 

```suggestion
// devCmd is the command for playground-style development
var devCmd = man.Docs.GetCommand("dev")
```

File: cmd/dev.go [hunk 2]
```diff
@@ -115,13 +122,9 @@ func injectLabelFlags(cmd *cobra.Command, isUpdate bool) {
...
 	designCmd := man.Docs.GetCommand("dev/design-system",
 		man.WithRun(dev_designSystem),
 	)
-
-	cmd := man.Docs.GetCommand("dev",
-		man.WithSubcommands(designCmd),
-	)
-	rootCmd.AddCommand(&cmd.Command)
+	devCmd.AddCommand(&designCmd.Command)
+	rootCmd.AddCommand(devCmd)
```
Comment (jrschumacher) [inline cmd/dev.go:129]:
```suggestion
	rootCmd.AddCommand(&devCmd.Command)
```

File: pkg/cli/table.go [hunk 3]
```diff
@@ -7,7 +7,7 @@ import (
 
 type Table table.Table
 
-var defaultTableWidth = 120
+var defaultTableWidth = 180
```
Comment (jrschumacher) [inline pkg/cli/table.go:10]: We can detect the size of the terminal with https://cs.opensource.google/go/x/term/+/v0.19.0:term.go;l=51

**Review by jrschumacher — CHANGES_REQUESTED** [review]
So only change I'm requesting is to use GetCommand. I've set GetDoc as deprecated. 

Otherwise some minor changes that could make your future life easier.

File: cmd/dev-selectors.go [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+	)
+	testCmd.Flags().StringArrayVarP(
+		&selectors,
+		testCmd.GetDocFlag("selector").Name,
+		testCmd.GetDocFlag("selector").Shorthand,
+		[]string{},
+		testCmd.GetDocFlag("selector").Description,
+	)
+
+	dev_selectorsCmd.AddSubcommands(genCmd, testCmd)
```
Comment (jrschumacher) [inline cmd/dev-selectors.go:147]: We offer two ways of doing this. One with a WithSubcommands and the other like this. I had to add this one due to a race condition. 

I'd like to settle on a unified approach to ensure it's easy to onboard new developers.

**Review by jrschumacher — CHANGES_REQUESTED** [review]

File: pkg/cli/table.go (hunk 3 above)
Comment (jakedoublev) [reply pkg/cli/table.go]: Updated with breakpoints.

File: cmd/dev-selectors.go (hunk 4 above)
Comment (jakedoublev) [reply cmd/dev-selectors.go]: Updated to `WithSubcommands`.

**Review by jrschumacher — APPROVED** [review]

**Pull request merged**